js/audio/Audio.stories.svelte

```diff
@@ -30,3 +30,13 @@
 		label: "Audio Recorder"
 	}}
 />
+
+<Story
```

Comment:
🥳

Reply:
Does the image already do this? Is this not something that `Block` in atoms can solve?

Reply:
Wrong comment.